api/test/nostd/string_view_test.cc

```diff
@@ -63,8 +63,10 @@ TEST(StringViewTest, SubstrPortion)
   EXPECT_EQ("12", s.substr(3, 2));
 }
 
+#if __EXCEPTIONS
 TEST(StringViewTest, SubstrOutOfRange)
 {
   string_view s = "abc123";
   EXPECT_THROW(s.substr(10), std::out_of_range);
```

Comment:
Can we do
```
#if __EXCEPTIONS
EXPECT_THROW(...)
#else
EXPECT_DEATH(...)
#endif
```
so that we still get the test coverage

Reply:
Good call! Done.